pkgs/applications/version-management/sourcehut/dispatch.nix

```diff
@@ -1,15 +1,15 @@
-{ lib, fetchgit, buildPythonPackage
+{ stdenv, lib, fetchgit, buildPythonPackage
```

Comment:
Please don't resurrect `stdenv` references. If they are necessary, they should be in this list in the order it appears (and `lib` is always first).

Reply:
Thx